addons/humanizer/scripts/resources/mhclo.gd

```diff
@@ -106,23 +107,39 @@ func calculate_mhclo_scale(helper_vertex_array: Array) -> Vector3:
 	var mhclo_scale = Vector3.ZERO
 	for axis in ["x","y","z"]:
 		var scale_data = scale_config[axis]
+		if is_zero_approx(scale_data.length):
+			#print("no scale was found, scale set to 1 for axis", axis)
+			mhclo_scale[axis] = 0.1 # im not sure why this magic number is correct for Fem_suit but it is...
+			continue # we're assuming a default
 		var start_coords = helper_vertex_array[scale_data.start]
 		var end_coords = helper_vertex_array[scale_data.end]
 		var basemesh_dist = absf(end_coords[axis] - start_coords[axis])
-		mhclo_scale[axis] = basemesh_dist/scale_data.length
-	return mhclo_scale
+		mhclo_scale[axis] = basemesh_dist / scale_data.length
```

Comment:
it's important here to always divide by anything other than 0.

This was causing nan's